lua/super-commit/config/user-config.lua

```diff
@@ -0,0 +1,15 @@
+local M = {}
+
+local config = {
+  debug = false
```

Comment:
`debug` is a reserved word, should not be here.
```suggestion
  verbose = false
```